packages/graphqlsp/src/checkImports.ts

```diff
@@ -148,8 +149,23 @@ function getFragmentsInSource(
 ): Array<FragmentDefinitionNode> {
   let fragments: Array<FragmentDefinitionNode> = [];
   const callExpressions = findAllCallExpressions(src, info, false);
+  const nodes = callExpressions.nodes.filter(x => {
```

Comment:
This doesn't quite look right 🤔 
It's impossible to link to TS’ `checker.ts` file on GitHub for this, since it's too long, but there's `checker.getExportSymbolOfSymbol` which can resolve `symbol.exportSymbol || symbol`, i.e.

```ts
getExportSymbolOfSymbol(symbol) {
    return getMergedSymbol(symbol.exportSymbol || symbol);
}
```

Then that can be filtered to check whether that symbol is exported, i.e.

```ts
function getExportSymbolOfValueSymbolIfExported(symbol: Symbol | undefined): Symbol | undefined {
    return getMergedSymbol(symbol && (symbol.flags & SymbolFlags.ExportValue) !== 0 && symbol.exportSymbol || symbol);
}
```

i.e. given the variable statement, we can extract the symbol then check:

```ts
const exportSymbol = checker.getExportSymbolOfSymbol(
  checker.getSymbolAtLocation(variableStatement)
);

const isExported = !!(symbol.flags & SymbolFlags.ExportValue);
```

Reply:
We are not using symbols though 😅